src/yandex_cloud_ml_sdk/_files/domain.py

```diff
@@ -32,6 +35,24 @@ async def _upload_bytes(
         expiration_policy: UndefinedOr[ExpirationPolicyAlias] = UNDEFINED,
         timeout: float = 60,
     ) -> FileTypeT:
+        """Uploads a byte array as a file.
+
+        :param data: The byte data to upload.
+        :param name: The name of the file.
+            Defaults to UNDEFINED.
+        :param description: A description of the file.
+            Defaults to UNDEFINED.
```

Comment:
Yeah, this is not interesting, that is defaults to UNDEFINED.

But what is really interesting, that by default server will try to auto-detect mime-type and you could override this. (I'm not sure actually how it works at server, I'm only supposing)